src/debugCodeLensProvider.ts

```diff
@@ -219,6 +219,7 @@ async function addToClasspath(uri: vscode.Uri): Promise<boolean> {
     if (ans === "Skip") {
         return true;
     } else if (ans === "Add to Source Path") {
+        sendInfo("", {operationName: "addToSourcePath"});
```

Comment:
It might be better to record each occurrence of `notOnClasspath` before Line 217 as well.

`sendInfo("", { operationName: "notOnClasspath" })`

Reply:
Will

```
sendInfo(operationId, {
            buildErrorTypes: “otnOnClasspath”,
            buildErrorCount: 1,
        });
```

better?